conf/routes

```diff
@@ -111,5 +102,9 @@ POST   /admin/v1/backfill/:name/start    @controllers.BackfillController.start(n
 POST   /admin/v1/users          @controllers.UserManagementController.createUser
 DELETE /admin/v1/users          @controllers.UserManagementController.deleteUser(email: java.lang.String ?= null)
 
+# Worker - Health Data
+POST /worker/v2/healthdata @controllers.HealthDataController.createRecord
```

Comment:
Similarly, this was pushed off to the next code change to keep the code review small. This is tracked in https://sagebionetworks.jira.com/browse/BRIDGE-364